src/widgets/Dialogs.js

```diff
@@ -180,7 +180,7 @@ define(function (require, exports, module) {
             }
         } else if (brackets.platform === "mac") {
             // CMD+D Don't Save
```

Comment:
Yeah, at least with the keyboards that I own from Apple that is the case. If I'm not mistaken, on older versions of Mac OS X 
the Don't Save action was bound to CMD+D... but not anymore :\



> On 01 Feb 2015, at 15:57, Marcel Gerber <notifications@github.com <mailto:notifications@github.com>> wrote:
> 
> In src/widgets/Dialogs.js <https://github.com/adobe/brackets/pull/10459#discussion_r23898065>:
> 
> > @@ -180,7 +180,7 @@ define(function (require, exports, module) {
> >              }
> >          } else if (brackets.platform === "mac") {
> >              // CMD+D Don't Save
> Ah ok, didn't know that. As stated above, I don't own a Macbook.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/adobe/brackets/pull/10459/files#r23898065>.
>